contracts/PinningManager.sol

```diff
@@ -250,7 +247,7 @@ contract PinningManager {
         //NO_OVERFLOW reasoning: see REF_DURATION
         require((request.startDate + request.numberOfPeriodsDeposited) * request.chosenPeriod <= now, "PinningManager: Request expired");
         //NO_OVERFLOW reasoning: chosenPrice is verified to not be zero in function call: newRequest
-        uint256 numberOfPeriodsDeposited = msg.value / request.chosenPrice;
+        uint256 numberOfPeriodsDeposited = (msg.value  request.size ) / request.chosenPrice;
```

Comment:
This does not look right, what about the operator? Or am I just blind? 😅
```suggestion
        uint256 numberOfPeriodsDeposited = (msg.value / request.size ) / request.chosenPrice;
```

Reply:
Nice catch Adam! Yeah I agree operator should be there.

Reply:
Oh no, indeed. I found this one myself too, but didn't push the change. Sorry guys